pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
+line-length = 90
+target-version = ['py38']
+include = '\.pyi?$'
+exclude = '''
+^/(
+  (
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
```

Comment:
Do we need to account for any other ignore files?  I'm thinking specifically of IDE stuff like .vscode.